reference/api/accountability/index.md

```diff
@@ -116,61 +125,13 @@ To add - see Issue [Accountability Contract Interface: add Usage and Examples to
 <!--
 
 ::: {.panel-tabset}
-## bash
```

Comment:
Yes. Now this is a different issue because this was again a product of the Quarto conversion from Hugo. That `## bash` should never have been there.

In Hugo for the Usage and Example tabs syntax is, using `aut node info` as an example:

>{{< tabpane langEqualsHeader=true >}}
>{{< tab header="aut" >}}
>aut node info
>{{< /tab >}}
>{{< /tabpane >}}

In Quarto this becomes:

>::: {.panel-tabset}
>## aut
>``` {.aut}
>aut node info
>```
>:::